manifests/o/ObinsKit/ObinsKit/1.2.10/ObinsKit.ObinsKit.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: ObinsKit.ObinsKit
```

Comment:
Why is the Publisher portion of the PackageId Obinskit? Shouldn't it be Hexcore, or whatever else the Publisher reports as

Reply:
blame wingetcreate https://github.com/microsoft/winget-pkgs/pull/17267 🤷‍♂️ Also a bunch of manifests have the PackageName as Publisher in the PackageID (Mostly from Manifestversion 0.0.1)

Where are you getting Hexcore from? Would you prefer `TaicangZhigengniaoInformationTechnology.ObinsKit` instead?

Reply:
I definitely like HexCore better than that :D I got it from the domain, and their about page: https://www.hexcore.xyz/about: 

> Hexcore is a brand founded by a group of designers and programmers full of dreams, dedicated to become an excellent peripherals company.

That said, I don't feel super strongly about it if you'd rather not change it

Reply:
Well, this is a new version PR, so we would have to move the Package.

Reply:
Yeah, perhaps not worth doing, feel free to resolve